app/src/pages/playground/SavePromptForm.tsx

```diff
@@ -133,32 +141,37 @@ export function SavePromptForm({
         />
       </View>
       <Form onSubmit={handleSubmit(onSubmit)}>
-        <View padding="size-200" paddingTop={0}>
+        <View paddingX="size-200" paddingBottom="size-200">
           <Controller
             name="description"
             control={control}
             render={({
               field: { onChange, onBlur, value },
               fieldState: { invalid, error },
             }) => (
-              <TextArea
-                label="Description"
-                description={
-                  mode === "create"
-                    ? "A description of your prompt (optional)"
-                    : "A description of your changes to the prompt (optional)"
-                }
-                isRequired={false}
-                height={100}
-                errorMessage={error?.message}
-                validationState={invalid ? "invalid" : "valid"}
+              <TextField
+                isInvalid={invalid}
                 onChange={onChange}
                 onBlur={onBlur}
                 value={value}
-              />
+                size="S"
+              >
+                <Label>Description</Label>
+                <TextArea />
+                {error ? (
+                  <FieldError>{error.message}</FieldError>
+                ) : (
+                  <Text slot="description">
+                    {mode === "create"
+                      ? "A description of your prompt (optional)"
+                      : "A description of your changes to the prompt (optional)"}
+                  </Text>
+                )}
+              </TextField>
```

Comment:
This is somewhat unrelated to your changes but, what do you think about changing the label between "Prompt Description" and "Change Description" or something like that?

It is kind of confusing that description can mean both things depending on if you are updating or creating

Reply:
agreed

Reply:
changed